src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/index.js

```diff
@@ -88,10 +88,18 @@ const AnalyzeAreasContainer = (props) => {
   }
 
   const onShapeUploadError = (error) => {
-    setPromptModalContent({
-      title: WARNING_MESSAGES[error.details.httpStatus].title,
-      description: WARNING_MESSAGES[error.details.httpStatus].description(),
-    });
+    if (error.message === "Invalid file format.") {
+      setPromptModalContent({
+        title: WARNING_MESSAGES.file.title,
+        description: WARNING_MESSAGES.file.description(),
+      });
+    }
+    if (error.message !== "Invalid file format.") {
+      setPromptModalContent({
+        title: WARNING_MESSAGES[error.details.httpStatus].title,
```

Comment:
We could also display the original error message to avoid wrong error messages like this one